README.md

```diff
@@ -5,103 +5,111 @@
 
 [Augury](https://augury.angular.io/) is a Google Chrome Dev Tools extension for debugging Angular 2 applications. You can install the extension from [Chrome Store](https://chrome.google.com/webstore/detail/augury/elgalmkoelokbchhkhacckoklkejnhcd).
 
-Once the extenion is installed you can test it against the demo application https://augury.angular.io/demo/
 
-![Screenshot of Augury](assets/screenloop.gif)
+## Supported version
 
-## Supported Version
+Augury works with application built starting with Angular 2.0.
 
-Currently works with applications built in [Angular 2.0.0](https://github.com/angular/angular/blob/master/CHANGELOG.md#200-2016-09-14) using the Angular Component Router version `3.0.0-beta.2`.
+# Known issues
```

Comment:
the end sounds good. not to complicated things but we can probably add a table of contents at the top? I think it could be auto-generated as well. Maybe in another PR or something